07-IO Streams/IOStreams/TestTasks.cs

```diff
@@ -7,34 +7,57 @@
 using System.Net;
 using System.Security.Cryptography;
 using System.Text;
+using System.Threading;
 using System.Xml.Linq;
 
 namespace IOStreams
 {
 
 	public static class TestTasks
 	{
-		/// <summary>
-		/// Parses Resourses\Planets.xlsx file and returns the planet data: 
-		///   Jupiter     69911.00
-		///   Saturn      58232.00
-		///   Uranus      25362.00
-		///    ...
-		/// See Resourses\Planets.xlsx for details
-		/// </summary>
-		/// <param name="xlsxFileName">source file name</param>
-		/// <returns>sequence of PlanetInfo</returns>
-		public static IEnumerable<PlanetInfo> ReadPlanetInfoFromXlsx(string xlsxFileName)
-		{
-			// TODO : Implement ReadPlanetInfoFromXlsx method using System.IO.Packaging + Linq-2-Xml
-
-			// HINT : Please be as simple & clear as possible.
-			//        No complex and common use cases, just this specified file.
-			//        Required data are stored in Planets.xlsx archive in 2 files:
-			//         /xl/sharedStrings.xml      - dictionary of all string values
-			//         /xl/worksheets/sheet1.xml  - main worksheet
+        /// <summary>
+        /// Parses Resourses\Planets.xlsx file and returns the planet data: 
+        ///   Jupiter     69911.00
+        ///   Saturn      58232.00
+        ///   Uranus      25362.00
+        ///    ...
+        /// See Resourses\Planets.xlsx for details
+        /// </summary>
+        /// <param name="xlsxFileName">source file name</param>
+        /// <returns>sequence of PlanetInfo</returns>
+        public static IEnumerable<PlanetInfo> ReadPlanetInfoFromXlsx(string xlsxFileName)
+        {
+            // TODO : Implement ReadPlanetInfoFromXlsx method using System.IO.Packaging + Linq-2-Xml
+
+            // HINT : Please be as simple & clear as possible.
+            //        No complex and common use cases, just this specified file.
+            //        Required data are stored in Planets.xlsx archive in 2 files:
+            //         /xl/sharedStrings.xml      - dictionary of all string values
+            //         /xl/worksheets/sheet1.xml  - main worksheet
+
+            //throw new NotImplementedException();
+
+            using (var planetsFile = new FileStream(xlsxFileName,FileMode.Open,FileAccess.Read))
+            {
+                Thread.CurrentThread.CurrentCulture = new System.Globalization.CultureInfo("en-US");
+                var zp = ZipPackage.Open(planetsFile);
```

Comment:
Read naming guidelines. And fix variable names
https://docs.microsoft.com/ru-ru/dotnet/standard/design-guidelines/general-naming-conventions